foundation-extensions/identity-for-edge-network/README.md

```diff
@@ -2,10 +2,10 @@
 
 The Adobe Experience Platform Identity mobile extension enables identity management from your mobile app when using the Adobe Experience Platform Mobile SDK and the [Edge Network extension](../experience-platform-extension/).
 
-## Configure the Adobe Experience Platform Identity extension in Experience Platform Launch
+## Configure the Adobe Experience Platform Identity extension in the Data Collection UI
```

Comment:
Make sure there is no usage for this anchor: `#configure-the-adobe-experience-platform-identity-extension-in-experience-platform-launch`

Reply:
Search the project, don't see any occasion is using this link.